compiler/luci-interpreter/include/luci_interpreter/Interpreter.h

```diff
@@ -60,11 +61,14 @@ class Interpreter
 
   void attachObserver(ExecutionObserver *observer);
 
+  inline Tensor *getTensorData(const loco::Node *node) { return _node_to_tensor[node]; }
```

Comment:
- `inline` is redundant here. All methods defined within class body are implicitly `inline`.
- Is it necessary to return non-constant `Tensor`?
- Since it now returns `Tensor`, I think the name of the method should be changed to reflect this.

Reply:
> inline is redundant here. All methods defined within class body are implicitly inline.

Thanks.

> Is it necessary to return non-constant Tensor?

No, it is not necessary yet. If we need such an interface, it will be possible to make a new function e.g, `getMutableTensor`.

> Since it now returns Tensor, I think the name of the method should be changed to reflect this.

I've changed the name to `getTensor`. Thanks.